### DISCUSSION TITLE
WalletClient type error

### DISCUSSION THREAD

**Discussion by branzinho** [wevm/wagmi#2730] [Q&A]
Trying the `getContract` example here with `walletClient` https://wagmi.sh/core/actions/getContract#walletclient-optional
However, getting a `Type 'GetWalletClientResult' is not assignable to type`... error.
![image](https://github.com/wagmi-dev/wagmi/assets/109865770/cfad9a24-7c62-42b8-ac0a-e104fe2de0f0)

I have the following versions:
"viem": "^1.2.14",
"wagmi": "^1.3.8"

**Comment by solomonNSI** [comment]
Were u able to solve it ser?
